README.md

```diff
@@ -1,3 +1,29 @@
 # hw01
 
-My name is Ahyeon Cho.
\ No newline at end of file
+# Brief biography 
```

Comment:
Awesome job! You included multiple headers, italics, bold, two kinds of lists, an embedded picture, and an embedded link - well beyond what the assignment required. I laughed for a full minute at "I do not like *steamed* carrots, but I do like *fresh* carrots" - it's an underrated but important distinction. I also just started playing Ocarina of Time!

Anyway - your image and links rendered correctly, and I know it took me a while to figure out how to get that to work, so you must've known to have the image in the same repo as the markdown file and to manually commit it. It's cool that you elaborated on why you want to take this course, and made the different parts of your bio distinct by using headers before each section. Also, great that you explained in your pull request message what aspects of this were difficult and what resources helped.

Reply:
I thought I would reply to this one since I would comment on the same aspects, largely. 
- I loved your use of two different types of headers. It made the final product look very streamlined. 
- Your italics were used to great effect with the carrots! 

On completely separate notes: I have heard so much about Ocarina of Time and am very sad I cannot currently play it...though maybe best for me right now! :) and I very much want to know your favorite 90s disney song. Mine is Make a Man out of You. In fact, I think I'll listen to that now.

Reply:
@ArlineLee  It's so good (Ocarina)!!

@ahyeoncho this prompted me to ask some friends about their carrot opinions and this was the result lmao:

![image](https://user-images.githubusercontent.com/56130450/79351113-e7436200-7efd-11ea-93a3-c7d170161eef.png)
![image](https://user-images.githubusercontent.com/56130450/79351139-f0343380-7efd-11ea-8104-28503db6bba3.png)

Reply:
@skanthan95 hahaha love it. I'm glad I'm not the only one that feels this way.

Reply:
@Arlinelee Make a man out of you is a classic! I also go back to Colors of the Wind haha